sunpy/net/attr.py

```diff
@@ -60,6 +113,55 @@ def collides(self, other):
     def __eq__(self, other):
         return dict(vars(self)) == dict(vars(other))
 
+    @classmethod
+    def update_values(cls, adict):
+        """
+        This is the method that clients will use to register their values for subclasses of `Attr`.
```

Comment:
This should have a more formatted docstring with `Parameters` and such. Also should follow the 
```
short description
<blank line>
long description
```

(because this method is public API)